packages/ui/app/src/docs/Header.tsx

```diff
@@ -4,6 +4,31 @@ import { HeaderButton } from "./HeaderButton";
 
 export const Header: React.FC = () => {
     const { resolveFile, docsDefinition } = useDocsContext();
+    const { logo, logoHref, navbarLinks } = docsDefinition.config;
+
+    const hasLogo = !!logo;
+    const hasLogoHref = !!logoHref;
```

Comment:
nit but let's just always do `!= null` as this can hide some weird bugs. e.g. `""` is coerced to `false` when doing `!!`

Reply:
Yeah in general I've been trying to fail fast - e.g. if we're storing urls as `""` that means something is probably wrong upstream and don't wanna swallow it.